website/static/playground.js

```diff
@@ -370,3 +352,34 @@ function showTooltip(elem, text) {
     elem.removeChild(tooltip);
   }, 2000);
 }
+
+function getExample(parser) {
+  switch (parser) {
+    case "babylon":
+      return [
+        'function HelloWorld({greeting = "hello", greeted = \'"World"\', silent = false, onMouseOver,}) {',
+        "",
+        "  if(!greeting){return null};",
+        "",
+        "     // TODO: Don't use random in render",
+        '  let num = Math.floor (Math.random() * 1E+7).toString().replace(/\\.\\d+/ig, "")',
+        "",
+        "  return <div className='HelloWorld' title={`You are visitor number ${ num }`} onMouseOver={onMouseOver}>",
+        "",
+        "    <strong>{ greeting.slice( 0, 1 ).toUpperCase() + greeting.slice(1).toLowerCase() }</strong>",
+        '    {greeting.endsWith(",") ? " " : <span style={{color: \'\\grey\'}}>", "</span> }',
+        "    <em>",
+        "\t{ greeted }",
+        "\t</em>",
+        "    { (silent)",
+        '      ? "."',
+        '      : "!"}',
+        "",
+        "    </div>;",
+        "",
+        "}"
+      ].join("\n");
+    default:
+      return "";
```

Comment:
Feel free to push examples to this PR directly as I'm not good at making examples. 😅